src/modules/Elsa.Dapper/Modules/Identity/Stores/RoleStore.cs

```diff
@@ -75,7 +63,8 @@ private RoleRecord Map(Role source)
         {
             Id = source.Id,
             Name = source.Name,
-            Permissions = string.Join(',', source.Permissions)
+            Permissions = string.Join(',', source.Permissions),
+            TenantId = source.TenantId
```

Comment:
Yes, this is already the case, actually. After the mapping, before the record gets saved, the tenant ID gets overwritten with the actual tenant ID, regardless of what is mapped here. To that end, we should remove this tenant ID mapping, to avoid confusion.